locale/pt-br/download/package-manager.md

```diff
@@ -0,0 +1,239 @@
+---
+layout: page.hbs
+title: Instalando Node.js via gerenciador pacotes
+---
+
+# Instalando Node.js via gerenciador pacotes
+
+***Nota:*** Os pacotes nessa página são mantidos e suportados por seus respectivos empacotadores, **não** pela equipe principal do Node.js. Por favor, relate quaisquer problemas encontrados para o mantenedor do pacote.
+
+----------------------------
+
+* [Android](#android)
+* [Arch Linux](#arch-linux)
+* [Debian e Destribuições Linux baseadas em Ubuntu, Enterprise Linux/Fedora e pacotes Snap](#debian-and-ubuntu-based-linux-distributions-enterprise-linux-fedora-and-snap-packages)
+* [FreeBSD](#freebsd)
+* [Gentoo](#gentoo)
+* [NetBSD](#netbsd)
+* [nvm](#nvm)
+* [OpenBSD](#openbsd)
+* [openSUSE e SLE](#opensuse-and-sle)
+* [macOS](#macos)
+* [SmartOS e illumos](#smartos-and-illumos)
+* [Solus](#solus)
+* [Void Linux](#void-linux)
+* [Windows](#windows)
+
+----------------------------
+
+## Android
+
+O suporte ao Android ainda é experimental no Node.js, portanto, os binários pré-compilados ainda não são fornecidos pelos programadores do Node.js
+
+Entretanto, existem algumas soluções de terceiros. Por exemplo, a comunidade [Termux](https://termux.com/) que fornece um emulador de terminal e ambiente Linux para Android, assim como seu próprio gerenciador de pacotes e [extensa coleção](https://github.com/termux/termux-packages) com muitas aplicações pré-compiladas. Esse comando no Termux vai instalar a última versão disponível do Node.js:
+
+```bash
+pkg install nodejs
+```
+
+Atualmente, os binários do Node.js no Termux estão ligados ao `system-icu` (dependência do pacote `libicu`)
+
+## Arch Linux
+
+O Node.js pacotes npm estão disponíveis no Repositório da Comunidade.
+
+```bash
+pacman -S nodejs npm
+```
+
+## Debian e distribuições Linux basedas em  Ubuntu, Enterprise Linux/Fedora e pacotes Snap
+
+[Distribuição dos binários oficiais do Node.js](https://github.com/nodesource/distributions/blob/master/README.md) são fornecidos pelo NodeSource.
+
+## FreeBSD
+
+A versão mais recente do Node.js está disponível na porta [www/node](http://freshports.org/www/node).
```

Comment:
The translation of bsb port to `porta` here feels weird, but I don't know any better translation